bson-adt-casbah/src/main/scala/adt/bson/casbah/CasbahImplicits.scala

```diff
@@ -24,7 +27,7 @@ trait CasbahImplicits {
 
   implicit def toDBObjectBsonOps(dbo: MongoDBObject): DBObjectBsonOps = new DBObjectBsonOps(dbo)
 
-  implicit def toDBObjectBsonOps(dbo: DBObject): DBObjectBsonOps = new DBObjectBsonOps(dbo)
+  implicit def toDBObjectBsonOps(dbo: com.mongodb.DBObject): DBObjectBsonOps = new DBObjectBsonOps(dbo)
```

Comment:
There is a `DBObject` extractor in the same package. It used to work fine because DBObject wasn't in the same package. When I moved this stuff into the Casbah package, it became a namespace conflict.